### ISSUE TITLE
Configure requisitions

### ISSUE THREAD

**Issue by devynspencer** [devynspencer/ansible-opennms#10]

**Comment by devynspencer** [comment]
@jeffgdotorg - I found [this section](http://docs.opennms.org/opennms/releases/17.1.1/guide-admin/guide-admin.html#_integrating_with_provisiond) in our Administrator's Guide while reading up on `provision.pl`, and it looks promising.

Provided that successive calls to the ReST api are idempotent, moving this into another bit of Python could be a good way to create an Ansible module for managing requisitions ( the first of many, hopefully ;) ).

**Comment by jeffgdotorg** [comment]
Yes, they are idempotent. When we spoke yesterday I assumed that you understood that provision.pl drives a ReST service; that's my bad.

I was thinking before I read this that operating the requisitions ReST service from Ansible ("push" mode of requisition-based provisioning) will probably be the quickest way forward. I can help you realize that.
